.github/workflows/ci.yml

```diff
@@ -103,7 +103,7 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        python-version: ["3.6", "3.7", "3.8", "3.9"]
+        python-version: ["3.7", "3.8", "3.9"]
         nautobot-version: ["1.1.5"]
```

Comment:
Note that CI is *not* actually testing nautobot 1.1.5 at present, rather it's testing the version we have in `poetry.lock` which is now 1.3.7 - see https://github.com/nautobot/nautobot-plugin-circuit-maintenance/runs/7265693745?check_suite_focus=true#step:5:339 for example.

If we want CI to actually test the specified Nautobot version (and we do!) then we need to ensure that our Dockerfile does a `poetry add nautobot==$NAUTOBOT_VER` before it does a `poetry install`.